basis/compiler/cfg/stacks/clearing/clearing.factor

```diff
@@ -1,11 +1,10 @@
 USING: accessors arrays assocs combinators.short-circuit
 compiler.cfg.instructions compiler.cfg.registers compiler.cfg.rpo
-compiler.cfg.stacks.map kernel math sequences ;
+compiler.cfg.stacks compiler.cfg.stacks.padding kernel math sequences ;
 IN: compiler.cfg.stacks.clearing
 
 : state>replaces ( state -- replaces )
-    [ stack>vacant ] map first2
-    [ [ <ds-loc> ] map ] [ [ <rs-loc> ] map ] bi* append
+    [ stack>vacant ] map { ds-loc rs-loc } [ swap create-locs ] 2map concat
```

Comment:
Does the order of this matter, previously it would be all ds's before all rs's.  Now it is ds/rs alternating?